plugins/lua/logistics.lua

```diff
@@ -168,8 +168,8 @@ local function print_status()
     print((' - Items designated for trading: %5d'):format(global_sp_stats.total_trade))
     print((' - Items designated for dumping: %5d'):format(global_sp_stats.total_dump))
     print((' - Animals designated for train: %5d'):format(global_sp_stats.total_train))  -- TODO: fix to include all animals designated for training
-    print((' - Forbidden items             : %5d'):format(global_sp_stats.total_forbid)) -- TODO: this should exclude unreachable items and buildings
-    print((' - All (unforbidden) items     : %5d'):format(global_sp_stats.total_claim))  -- TODO: same ^^ implement in logistics.cpp or use item script api? (reqscript('item'))
+    print((' - Forbidden items             : %5d'):format(global_sp_stats.total_forbid)) -- TODO: this should exclude unreachable items
```

Comment:
these numbers only involve items in stockpiles. "reachable" isn't really a concern here, is it?

Reply:
Well, as it was previously written, I'm pretty sure it isn't limited to only items in stockpiles. I created the totals for the items in stockpiles and they don't match. Where you pointed out I could be using get_bad_flags for a bitwise compare, that's looping through all items in play to get total counts of melt designations.

I agree, this plugin isn't about summaries for all items... but it was there and makes sense in the context of, "what do I have designated for melt?" Extending it for, "what do I have designated as forbidden," causes the paradigm to break down because we often have so many items forbidden beyond what we care about. And that's where it's a bit odd for animals being trained as well... "wait, I thought I had more animals being trained! ...oh, I do, just not animals in stockpiles."

The more I thought about it the more I liked the summary's existing function to be a total of everything in the fort, but that's when I noted these as needing improvement. And secondarily, I thought it would be cool to show, "stockpiled" / "total" for each line, which would further clarify why this summary is there rather than just using the new totals in footer of the stockpiles table.

But I dunno... that's basically just my thought process. Maybe I'm overthinking it?

Reply:
![image](https://github.com/DFHack/dfhack/assets/150195674/9e698d60-4e44-43a2-966b-85b396d98848)